lib/spack/spack/architecture.py

```diff
@@ -214,7 +215,11 @@ def optimization_flags(self, compiler):
             import spack.spec
             if isinstance(compiler, spack.spec.CompilerSpec):
                 compiler = spack.compilers.compilers_for_spec(compiler).pop()
-            compiler_version = compiler.cc_version(compiler.cc)
+            try:
+                compiler_version = compiler.cc_version(compiler.cc)
+            except spack.util.executable.ProcessError as e:
```

Comment:
Does this mean that PGI on powerpc will use the verbatim version? That will cause problems in `archspec`

Reply:
No, because 2 is ignored. It’ll parse the output, ignore the error, and return the real version.  The ProcessError is only raised for errors we don’t ignore.